src/app/(pages)/craft/carousel/Carousel.tsx

```diff
@@ -87,7 +132,7 @@ export default function Carousel() {
             onClick={() => handleSlideChange(currentSlide - 1)}
             disabled={currentSlide === 0}
             className={c(
-              'mr-2 rounded-sm text-neutral-600 disabled:cursor-not-allowed dark:disabled:text-neutral-600 disabled:text-neutral-300 dark:border-neutral-800 dark:text-neutral-300 duration-150 hover:text-neutral-400 dark:hover:text-neutral-400'
+              'mr-2 rounded-sm text-neutral-600 hover:text-neutral-400 duration-150 dark:border-neutral-800 dark:text-neutral-300 dark:hover:text-neutral-400 disabled:cursor-not-allowed disabled:text-neutral-300 dark:disabled:text-neutral-600'
             )}
           >
             <ChevronLeftIcon width={25} height={25} aria-label="Left chevron icon" />
```

Comment:
Probably don't need this `aria-label` as the `<button />` has some screen reader text inside it.

Reply:
ahh ok, I get that – good spot, thank you!